webapp/src/dogma/features/repo/NewRepoToken.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  Button,
+  FormControl,
+  FormErrorMessage,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverCloseButton,
+  PopoverContent,
+  PopoverFooter,
+  PopoverHeader,
+  PopoverTrigger,
+  Radio,
+  RadioGroup,
+  Spacer,
+  Stack,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { Controller, useForm } from 'react-hook-form';
+import { IoMdArrowDropdown } from 'react-icons/io';
+import { useState } from 'react';
+import { useGetTokensQuery } from 'dogma/features/api/apiSlice';
+import { TokenDto } from 'dogma/features/token/TokenDto';
+import { OptionBase, Select } from 'chakra-react-select';
+import { ConfirmAddToken } from 'dogma/features/repo/ConfirmAddToken';
+
+interface TokenOptionType extends OptionBase {
+  value: string;
+  label: string;
+}
+
+type FormData = {
+  appId: string;
+  role: string;
+};
+
+export const NewRepoToken = ({ projectName }: { projectName: string }) => {
```

Comment:
`RepoToken` is not a term we commonly use and it is a token used at a project level.  How about `AppToken` although other tabs start with `Repo` otherwise `ProjectToken`?

```suggestion
export const NewAppToken = ({ projectName }: { projectName: string }) => {
```